FORMAT.md

```diff
@@ -135,9 +155,31 @@ The joint refers to a massless degree of freedom.
 <joint axis="rx"/>
 ```
 
+### `<end-effector>`
+
+An end effector refers to a component at the end of a kinematic chain (e.g., that has no children).  The key aspect of the end effector is that it identifies the location of an "end effector frame" at a specified relative position.
+
+(Note that HRDFs of version <= 1.1.0 do not explicitly have the notion of an end effector frame, so when being loaded into a compliant parser, the API adds an implicit "end effector frame" to the end of the chain of elements).
+
+**Required attributes:**
+
+- `type` (string/enum) Currently supported values:
+  - custom (fully specifiable by the user)
```

Comment:
maybe make this the default if unspecified?

I think `<end-effector />` should just add an identity transform to the previous body

Reply:
I would be OK with making this optional -- let me just pass that be people here before committing to it.

I think the center of the pads when closed is a reasonable location for the parallel output.